lib/config/options/index.ts

```diff
@@ -1247,6 +1247,20 @@ const options: RenovateOptions[] = [
     cli: false,
     mergeable: true,
   },
+  {
```

Comment:
are you overriding pinDigest object, but pinDigest worked before for other cases, wouldn't that cause regression on old cases that needed the old digest format? 
were there tests for pinDigest? that may have failed because of this? or is this the right thing to do?